os/include/tinyara/sched.h

```diff
@@ -553,6 +553,9 @@ struct tcb_s {
 #ifdef CONFIG_SIGKILL_HANDLER
 	_sa_sigaction_t sigkillusrhandler; /* User defined SIGKILL handler      */
 #endif
+#ifdef HAVE_GROUP_MEMBERS
+	sigset_t sighandlemask;		/* Signals that are blocked for handling */
```

Comment:
1. How about `sigreceivemask`?
2. `for handling` is not understandable exactly. Let's rewrite it.